### PR TITLE
Update config-settings.rst

### PR DESCRIPTION
**Pull request by jasonblais**
Includes 
 - Fixed minor typo
 - New setting: LoggingSettings > Enable Webhook Debugging
 - New setting: Notifications > Notification Footer Address
 - Updated setting: LocalizationSettings > AvailableLocales

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -60,10 +60,11 @@ Default Client Language ``"DefaultClientLocale": "en"``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Default language for newly created users and pages where the user hasn't logged in.
 
-Available Languages ``"AvailableLocales": "en,es,fr,ja,pt-BR"``
+Available Languages ``"AvailableLocales": ""``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  
-Sets which languages are available for users in **Account Settings** > **Display** > **Languages**. Add new languages using the dropdown menu as they become available.
+Sets which languages are available for users in **Account Settings** > **Display** > **Languages**. Leave the field blank to add new languages automatically by default, or add new languages using the dropdown menu manually as they become available.
```
Comment (esethna) [inline source/administration/config-settings.rst]: We should add a note here that if  you're setting them manually, the default client language has to be selected here or else you can't save the setting

**Comment by jasonblais** [comment]
Updated based on Eric's feedback

**Comment by esethna** [comment]
+1 added some other minor tweaks

**Comment by esethna** [comment]
Merging this to avoid conflicts with another PR to be submitted.

**Pull request merged**